packages/arb-token-bridge-ui/src/components/TransferPanel/TokenButton.tsx

```diff
@@ -84,7 +108,9 @@ export function TokenButton({
                   <Loader size="small" color="white" />
                 ) : (
                   <>
-                    <TokenLogo srcOverride={options?.logoSrc} />
+                    <TokenLogo
+                      srcOverride={options?.logoSrc ?? tokenLogoSrcOverride}
```

Comment:
We already have `options?.logoSrc` handled in `tokenLogoSrcOverride` so this can simply be `srcOverride={tokenLogoSrcOverride}`